components/aws/common/common.mjs

```diff
@@ -0,0 +1,124 @@
+import { EC2Client } from "@aws-sdk/client-ec2";
+import { LambdaClient } from "@aws-sdk/client-lambda";
+import { SSMClient } from "@aws-sdk/client-ssm";
+import { IAMClient } from "@aws-sdk/client-iam";
+import { S3Client } from "@aws-sdk/client-s3";
+import { CloudWatchLogsClient } from "@aws-sdk/client-cloudwatch-logs";
+import { EventBridgeClient } from "@aws-sdk/client-eventbridge";
+import { SQSClient } from "@aws-sdk/client-sqs";
+import { SNSClient } from "@aws-sdk/client-sns";
+import { SESClient } from "@aws-sdk/client-ses";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+
+export default {
+  defaultRegion: "us-east-1",
+  awsClients: {
+    ec2: EC2Client,
+    lambda: LambdaClient,
+    ssm: SSMClient,
+    iam: IAMClient,
+    s3: S3Client,
+    cloudWatchLogs: CloudWatchLogsClient,
+    eventBridge: EventBridgeClient,
+    sqs: SQSClient,
+    sns: SNSClient,
+    ses: SESClient,
```

Comment:
For some reason if `SESClient` is kept, when publishing any action pipedream api will return http 500

Reply:
I see the same error when publishing components. Removing several of the other awsClients also fixes the 500 error for me. I think the issue might be that each component's bundle size is too large to publish with all of the packages included.

Reply:
Yeah, I tried that and got the same results. What should we do in this case?

Reply:
To reduce the bundle size, we could remove the (component-specific) AWS SDK client imports (and methods) from the app file–which is imported in every component–and import them in only components that use that client. And we could create a common file for each AWS service/client like the [common files](https://github.com/andrewjschuang/pipedream/tree/0081cdbe9a6cc3ede53218994cf0a368665b132e/components/aws/sources/common) for the AWS sources. The common files would import the SDK packages and instantiate the clients using a method in the app file. Let me know if you think that'll work.